src/Orchard.DisplayManagement/Descriptors/DefaultShapeTableManager.cs

```diff
@@ -140,6 +105,64 @@ public ShapeTable GetShapeTable(string themeName)
             return shapeTable;
         }
 
+        private IEnumerable<ShapeAlteration> BuildAlterations(IShapeTableProvider bindingStrategy, Feature strategyFeature, IList<string> excludedFeatures)
+        {
+            IEnumerable<ShapeAlteration> builtAlterations = null;
```

Comment:
Remove, it has to be the result of the Lazy evaluation that is returned directly